src/controllers/roles.js

```diff
@@ -1,225 +1,247 @@
-import models from '../database/models';
+import models from '../database/models'
 const users = async (req, res) => {
-  try {
-    const users = await models.User.findAll();
-    return res.status(200).json({
-      message: 'Users',
-      count: users.length,
-      response: users,
-    });
-  } catch (error) {
-    return res.status(500).json({ error: error.message });
-  }
-};
+    try {
+        const users = await models.User.findAll({
+            attributes: { exclude: ['password', 'email'] },
```

Comment:
Was it necessary to remove the email also?

Reply:
yes for user security purpose